frontend/src/lib/components/PropertyFilters/PropertyFilters.js

```diff
@@ -95,24 +95,28 @@ export function PropertyFilters({
     const logic = propertyFilterLogic({ propertyFilters, endpoint, onChange, pageKey })
     const { filters } = useValues(logic)
 
+    const hiddenFilters = ['test_environment']
```

Comment:
I'd suggest using the data in PropertyKeyInfo.tsx here instead of this random array